templates/install.tmpl

```diff
@@ -307,4 +307,20 @@
 		</div>
 	</div>
 </div>
+<div class="hidden">
+	<script type="text/javascript">
+		<!--//--><![CDATA[//><!--
+			var images = new Array()
+			function preload() {
+				for (i = 0; i < preload.arguments.length; i++) {
+					images[i] = new Image()
+					images[i].src = preload.arguments[i]
+				}
+			}
+			preload(
+				"{{StaticUrlPrefix}}/img/loading.png"
+			)
+		//--><!]]>
+	</script>
+</div>
```

Comment:
Can you extract this to a standalone JS file similar to how it's done for Swagger, e.g. create webpack chunk and load it:

````js
<script src="{{StaticUrlPrefix}}/js/install.js"></script>
````

Reply:
Thought the usefulness of this script kind of eludes me. It also seems to be written in a syntax from ~15 years go.

Reply:
I knew you were going to say that. The idea is simply to precache the loading gif so that it shows when the interstitial page is flashed. The problem is that by the time the user gets the interstitial page and then requests the loading.gif the macaron may be down.

Reply:
It's just the fact that we have unlinted JS here bothers me (and that the code is overly complex). How about one of these methods?

````html
<link rel="prefetch" href="{{StaticUrlPrefix}}/img/loading.png" />
````
````html
<img style="display: none" src="{{StaticUrlPrefix}}/img/loading.png"/>
````

Reply:
Actually we can use the (proper) helper class:

```html
<img class="hide" src="{{StaticUrlPrefix}}/img/loading.png"/>

Reply:
Turns out we don't really have a propper "hidden" helper (one with `!important`), so this inline style is fine I guess.